packages/pg/pg-server-tests.js

```diff
@@ -0,0 +1,65 @@
+Table = null; //                                          All being well we'll overwrite this later
+const tablename = 'testing'; //                           Pick a sensible (?) table name
+
+Tinytest.add('pg - schema builder - create testing table', (test) => {
+    let dropTable = () => { //                            Set up a promised drop table
```

Comment:
Use function declarations. Here's the new styleguide we are working on: https://github.com/meteor/javascript